src/frontend/src/catalog/system_catalog/rw_catalog/rw_actors.rs

```diff
@@ -12,15 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use risingwave_common::types::DataType;
+use std::sync::LazyLock;
 
-use crate::catalog::system_catalog::SystemCatalogColumnsDef;
+use itertools::Itertools;
+use risingwave_common::catalog::RW_CATALOG_SCHEMA_NAME;
+use risingwave_common::error::Result;
+use risingwave_common::row::OwnedRow;
+use risingwave_common::types::{DataType, ScalarImpl};
 
-pub const RW_ACTORS_TABLE_NAME: &str = "rw_actors";
+use crate::catalog::system_catalog::{BuiltinTable, SysCatalogReaderImpl};
 
-pub const RW_ACTORS_COLUMNS: &[SystemCatalogColumnsDef<'_>] = &[
-    (DataType::Int32, "actor_id"),
-    (DataType::Int32, "fragment_id"),
-    (DataType::Int32, "parallel_unit_id"),
-    (DataType::Varchar, "status"),
-];
+pub static RW_ACTORS: LazyLock<BuiltinTable> = LazyLock::new(|| BuiltinTable {
```

Comment:
I guess we don't need `LazyLock` for `BuiltinTable`? Its construction can be evaluated constantly.

Reply:
👍 No for `rw_actors`, but I think we might still need it for some `BuiltinTable`s with columns in `DataType::ListBox<DataType>`.